### ISSUE TITLE
Various warnings after reloading VS Code, and eventually unable to reconnect

### ISSUE THREAD

**Issue by bamurtaugh** [Azure/azure-functions-vscode-web#23]
I ran command `Developer: Reload Window`, and now I get:

![Image](https://github.com/user-attachments/assets/710e6ce6-e5a4-46f7-9a0f-d050be719469)

**Comment by bamurtaugh** [comment]
I tried reloading another time, and I got a new error (but not the one above interestingly):

![Image](https://github.com/user-attachments/assets/823b2cff-4fe1-42fe-826b-efc296765a8a)

I also now have some unhappy terminals:

![Image](https://github.com/user-attachments/assets/ec19bb1d-8056-4cb4-9647-42d20457b0d6)

**Comment by bamurtaugh** [comment]
I reloaded a third time, and I got no warnings / errors this time, and the unhappy terminals seem to have gone away. But then I realized I wasn't able to reconnect:

![Image](https://github.com/user-attachments/assets/e55bc268-8c6a-490f-a0e2-df450c30750f)

I selected `Reload Window` and I can no longer reconnect. It's been several minutes, and I keep seeing `Provisioning session infra resources...` pop up, go away, pop up, etc.

![Image](https://github.com/user-attachments/assets/04edc65d-6c28-4849-b4c5-dc97690db7b1)

I tried reloading again (this time manually refreshing the browser, not via VS Code reload command), and I'm eventually able to get back in, but I've now lost all my work (as I think I'm connected to new compute?). 

![Image](https://github.com/user-attachments/assets/5806fd85-5bd0-4e2c-aa02-752f0e1ae6f6)

It would've been great to:
1) Get a better understanding why I was stuck reloading 
2) See a warning when I go to refresh that I'd lose all my work
